app/src/main/java/com/gleam/kiwi/model/TimeTableInfo.kt

```diff
@@ -9,41 +13,46 @@ data class TimeTableInfo(
 )
 
 data class Mon(
-    val `1`: String,
-    val `2`: String,
-    val `3`: String,
-    val `4`: String,
-    val `5`: String
+    val `1`: Details?,
+    val `2`: Details?,
+    val `3`: Details?,
+    val `4`: Details?,
+    val `5`: Details?
 )
 
 data class Tue(
-    val `1`: String,
-    val `2`: String,
-    val `3`: String,
-    val `4`: String,
-    val `5`: String
+    val `1`: Details?,
+    val `2`: Details?,
+    val `3`: Details?,
+    val `4`: Details?,
+    val `5`: Details?
 )
 
 data class Wed(
-    val `1`: String,
-    val `2`: String,
-    val `3`: String,
-    val `4`: String,
-    val `5`: String
+    val `1`: Details?,
+    val `2`: Details?,
+    val `3`: Details?,
+    val `4`: Details?,
+    val `5`: Details?
 )
 
 data class Thu(
-    val `1`: String,
-    val `2`: String,
-    val `3`: String,
-    val `4`: String,
-    val `5`: String
+    val `1`: Details?,
+    val `2`: Details?,
+    val `3`: Details?,
+    val `4`: Details?,
+    val `5`: Details?
 )
 
 data class Fri(
-    val `1`: String,
-    val `2`: String,
-    val `3`: String,
-    val `4`: String,
-    val `5`: String
+    val `1`: Details?,
+    val `2`: Details?,
+    val `3`: Details?,
+    val `4`: Details?,
+    val `5`: Details?
+)
+
+data class Details(
+    val subject: String,
+    val room: String
```

Comment:
roomをnullableに変更してください.
```suggestion
    val room: String?
```

Reply:
### https://github.com/the-gleam/kiwi-basket/blob/master/README.md

Reply:
UserInfo.ktとTiimeTableIinfo.ktの修正をしたので確認してください。  
https://github.com/the-gleam/kiwi/pull/3/commits/ad1f7bc90deef7da36ca2c384ee0b9322e4cd865

Reply:
確認しました. 大丈夫そうです.